composer.json

```diff
@@ -24,9 +24,11 @@
 		"jakub-onderka/php-parallel-lint": "^1.0",
 		"nette/application": "^2.3.0 || ^3.0.0",
 		"nette/component-model": "^2.3.0 || ^3.0.0",
+		"nette/database": "^2.3.0 || ^3.0",
 		"nette/di": "^2.3.0 || ^3.0.0",
 		"nette/forms": "^2.3.0 || ^3.0.0",
 		"nette/http": "^2.3.0 || ^3.0.0",
+		"nette/routing": "^3.0.0",
```

Comment:
There's no need for adding these in require-dev.

Reply:
Why not? We use them.

Reply:
Where?

Reply:
I think the build will pass if we remove all/most of them.

Reply:
They are used in stubs.

Reply:
As I've said a few days ago in the Slack thread, classes existing outside of stubs do not have any effect on stubs themselves.

Reply:
I removed nette/routing and build passes, but nette/database is used in that new generics test.